AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/Base Classes/ALHomeViewController.swift

```diff
@@ -13,9 +13,8 @@ import SafariServices
 
 class ALHomeViewController: UITableViewController
 {
-    let kSupportLink = "https://support.applovin.com/support/home"
-    
-    let kRowIndexToHideForPhones = 3;
+    let kSupportLink = "https://support.applovin.com/hc/en-us"
+    let kRowIndexToHideForPhones = 3
```

Comment:
Can prob make these static as well - there's no need for them to be instance

Reply:
Then you can reference them as `Self.kSupportLink`

Reply:
done @thomasmso